src/triage/component/catwalk/evaluation.py

```diff
@@ -110,72 +111,177 @@ def _validate_metrics(
                 raise ValueError("For custom metric {} greater_is_better must be "
                                  "boolean True or False".format(name))
 
-    def _generate_evaluations(
+    def _build_parameter_string(
+        self,
+        threshold_unit,
+        threshold_value,
+        parameter_combination
+    ):
+        """Encode the metric parameters and threshold into a short, human-parseable string
+
+        Examples are: '100_abs', '5_pct'
+
+        Args:
+            threshold_unit (string) the type of threshold, either 'percentile' or 'top_n'
+            threshold_value (int) the numeric threshold,
+            parameter_combination (dict) The non-threshold parameter keys and values used
```

Comment:
Given that the metrics dict has been split a number of times before this function, a small example might be nice just for readability. E.g,. right now I think that parameter_combination contains something of the form `{"alpha":13, "beta": 1.2}`